actors/migration/nv9/init.go

```diff
@@ -12,7 +12,7 @@ import (
 
 type initMigrator struct{}
 
-func (m initMigrator) MigrateState(ctx context.Context, store cbor.IpldStore, in StateMigrationInput) (*StateMigrationResult, error) {
+func (m initMigrator) MigrateState(ctx context.Context, store cbor.IpldStore, in actorMigrationInput) (*actorMigrationResult, error) {
```

Comment:
nit: making MigrateState public but `actorMigrationResult` private is strange. Given that this is all private anyways, I'd just make the function private too.

Reply:
Done